cypress/fixtures/README.md

```diff
@@ -27,135 +27,8 @@ Within the `cypress/fixtures` folder, we have the following sub-folders:
-| ------------------ | --------------------------------------------------------------------------------------------------------------------- | ------------------------------------ |
-| method or event    | Name of the API or Event, which is used to extract the response object from global list for content validation.       | accessibility.closedCaptionsSettings |
-| validationJsonPath | Path of the response value that needs to be validated                                                                 | result.enabled                       |
-| context            | Represent the data that needs to be stored in api/event object that helps to search specific object in a global list. | {}                                   |
-| content            | Represent the expected value used for validating against a firebolt api respo                                         | true                                 |
-| expectingError     | Determines whether we are performing result or error content validation.                                              | true                                 |
-
-##### Examples:
-
-**Example 1:**
-
-- FCS has default data file, which contains the data that can be used across feature files. Ex: `TRUE` value is present in [defaultData.json](./defaultTestData.json)
-
-```
-"ENABLE_AUDIODESCRIPTION_WITH_INTEGER_PARAMETER":{
-  "method": "accessibility.closedCaptionsSettings",
-  "context": "noContext",
-  "validationJsonPath": "result.enabled",
-  "content": "TRUE"
-}
-```
-
-**Example 2:**
-
-- FCS has default data file, which contains the data that can be used across feature files. Ex: `FALSE` value is present in [defaultData.json](./defaultTestData.json)
-
-```
-"ONCLOSEDCAPTIONSSETTINGSCHANGED_WITH_DISABLED": {
-  "event": "accessibility.onClosedCaptionsSettingsChanged",
-  "context": "noContext",
-  "validationJsonPath": "eventResponse.enabled",
-  "content": "FALSE"
-}
-```
-
-**Example 3:**
-
-- `ACCOUNT_ID` content object is present inside the `fixtures/objects/validationObjects/account.json` file.
-- `ACCOUNT_ID` is split into two parts: `ACCOUNT` and `ID`.
-- `ACCOUNT` refers to a file name, and `ID` is an object key name present inside the `account.json` file.
-
-```
-"ACCOUNT_ID": {
-  "method": "account.id",
-  "validationJsonPath": "result",
-  "content": "ACCOUNT_ID"
-}
-
-```
-
-**Note:** 
-The fields listed below are optional; if any of them are missing, default values listed below will be added during runtime. 
-```
-- context - {} 
-- params - {} 
-- validationJsonPath - "result" 
-- content - null 
-```
-If the firebolt validation object has a'method' field, it validates the method; otherwise, it validates the event.
-
+1. [Static JSON Objects](docs/staticJSONObjects.md)
+2. [Static JS Objects](docs/staticJSObjects.md)
+3. [Dynamic JSON Objects](docs/dynamicObjects.md)
```

Comment:
Line 34 is not Dynamic JSON Objects. It should be Dynamic JS Objects

Reply:
docs/staticJSObjects.md - spelling is wrong

Reply:
addressed the comment